lib/validators.js

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import {isDidUrl} from './did-io.js';
+import {DidResolverError} from './DidResolverError.js';
+
+/**
+ * Determines if a did contains url characters and
+ * then validates either the didUrl or did.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.did - A did.
+ *
+ * @throws {DidResolverError} Throws if did is invalid.
+ *
+ * @returns {undefined} Just returns on success.
```

Comment:
Does jsdoc lint require this?  If not could be omitted.  If so, take out the "Just".

Reply:
I checked and weirdly it does return a type on the return which is a little alarming. Took the other comment and removed the `Just` here: https://github.com/digitalbazaar/did-io/pull/63/commits/f1f6f696881158b19ee0378b60177a05e4855c57